programs/ssl/ssl_client2.c

```diff
@@ -534,6 +537,14 @@ int main( int argc, char *argv[] )
     char *p, *q;
     const int *list;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( mbedtls_platform_setup( &platform_ctx ) != 0 )
+    {
+        mbedtls_printf( "Failed initializing platform.\n" );
```

Comment:
Error message is inconsistent with other error messages in the app. e.g. `" failed\n  ! mbedtls_ssl_handshake returned -0x%x\n\n"` at line 1972.